content/security/for-admins/provisioning/group-mapping.md

```diff
@@ -0,0 +1,181 @@
+---
+description: Group mapping for administrators
+keywords: Group Mapping, SCIM, Docker Hub, Docker Admin, admin, security
+title: Group mapping
+aliases:
+- /admin/company/settings/group-mapping/
+- /admin/organization/security-settings/group-mapping/
+- /docker-hub/group-mapping/
+- /security/for-admins/group-mapping/
+---
+
+With directory group-to-team provisioning from your IdP, user updates will automatically sync with your Docker organizations and teams. You can use group mapping once you have configured [single sign-on (SSO)](../single-sign-on/_index.md).
+
+> **Tip**
+>
+> Group mapping is ideal for adding a user to multiple organizations or multiple teams within one organization. If you don't need to set up multi-organization or multi-team assignment, you can use [user-level attributes](scim.md#set-up-role-mapping).
+{ .tip }
+
+## How group mapping works
+
+IdPs share with Docker the main attributes of every authorized user through SSO, such as email address, name, surname, and groups. Just-In-Time (JIT) Provisioning uses these attributes to create or update the user’s Docker profile and their associations with organizations and teams on Docker Hub.
```

Comment:
I noticed that provisioning (and "in" in JIT) is not capitalized on the JIT page, so I guess it shouldn't be here either

```suggestion
IdPs share with Docker the main attributes of every authorized user through SSO, such as email address, name, surname, and groups. Just-in-Time (JIT) provisioning uses these attributes to create or update the user’s Docker profile and their associations with organizations and teams on Docker Hub.
```